src/shared/components/query/filteredSearch/field/CheckboxFilterField.tsx

```diff
@@ -58,7 +58,7 @@ export const FilterCheckbox: FunctionComponent<FieldProps> = props => {
             <h5>{props.filter.form.label}</h5>
             <div>
                 {options.map((option: FilterFieldOption) => {
-                    const id = `input-${option}`;
+                    const id = `input-${option.displayValue}`;
```

Comment:
Filter can now still be matched to the wrong checkbox, when their `displayValues` are the same. Maybe better to just wrap the checkbox by its label?
```js
                {options.map((option: FilterFieldOption) => {
                    let isChecked = checkedOptions.includes(option.value);
                    return (
                        <div
                            style={{
                                display: 'inline-block',
                                padding: '0 1em 0 0',
                            }}
                        >
                            <label
                                style={{
                                    display: 'inline-block',
                                    padding: '0 0 0 0.2em',
                                }}
                            >
                                <input
                                    type="checkbox"
                                    value={option.displayValue}
                                    checked={isChecked}
                                    onClick={() => {
                                        isChecked = !isChecked;
                                        updatePhrases(option.value, isChecked);
                                        const update = createQueryUpdate(
                                            toRemove,
                                            checkedOptions,
                                            props.filter
                                        );
                                        props.onChange(update);
                                    }}
                                    style={{
                                        display: 'inline-block',
                                    }}
                                />
                                {' '}
                                {option.displayValue}
                            </label>
                        </div>
                    );
                })}
                
```

Reply:
now, the id will consist of displayvalue and optionvalue combined to ensure that they are unique. Enwrapped with label.